From c6fe55cf6089fc5cf1cea15fc7e1c9a8b90d9fda Mon Sep 17 00:00:00 2001
From: Andreas Jaeger <aj@suse.de>
Date: Fri, 11 Jan 2013 11:53:13 +0100
Subject: [PATCH] Add MSG_FASTOPEN

	[BZ #15003]
	* sysdeps/unix/sysv/linux/bits/socket.h (MSG_FASTOPEN): New
	value. Sync with Linux 3.7.
---
 sysdeps/unix/sysv/linux/bits/socket.h | 2 ++
 2 files changed, 8 insertions(+)

diff --git a/sysdeps/unix/sysv/linux/bits/socket.h b/sysdeps/unix/sysv/linux/bits/socket.h
index 25b115e..eadd7d9 100644
--- a/sysdeps/unix/sysv/linux/bits/socket.h
+++ b/sysdeps/unix/sysv/linux/bits/socket.h
@@ -207,6 +207,8 @@ enum
 #define	MSG_MORE	MSG_MORE
     MSG_WAITFORONE	= 0x10000, /* Wait for at least one packet to return.*/
 #define MSG_WAITFORONE	MSG_WAITFORONE
+    MSG_FASTOPEN	= 0x20000000, /* Send data in TCP SYN.  */
+#define MSG_FASTOPEN	MSG_FASTOPEN
 
     MSG_CMSG_CLOEXEC	= 0x40000000	/* Set close_on_exit for file
 					   descriptor received through
-- 
1.8.1.2

